README.md

```diff
@@ -95,6 +95,77 @@ treeForAddon(tree) {
 }
 ```
 
+### Debug Tooling
+
+In order to allow apps and addons to easily provide good development mode ergonomics (assertions, deprecations, etc) but
+still perform well in production mode ember-cli-babel automatically manages stripping / removing certain debug
+statements. This concept was originally proposed in [ember-cli/rfcs#50](https://github.com/ember-cli/rfcs/pull/50), 
+but has been slightly modified during implementation (after researching what works well and what does not).
+
+#### Debug Macros
+
+To add convienient deprecations and assertions, consumers (in either an app or an addon) can do the following:
```

Comment:
can you add a bulleted list of available macros, or a link to where they are documented?